hello-world/use-plugin/src/test/kotlin/io/arrowkt/example/MetaPluginTest.kt

```diff
@@ -1,17 +1,16 @@
 package io.arrowkt.example
 
-import org.hamcrest.MatcherAssert.assertThat
-import org.hamcrest.core.IsEqual.equalTo
-import org.junit.Test
 import java.io.ByteArrayOutputStream
 import java.io.PrintStream
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions;
 
 class HelloWorldTest {
   @Test
   fun `greets ΛRROW Meta`() {
       val out = ByteArrayOutputStream()
       System.setOut(PrintStream(out))
       helloWorld()
-      assertThat(out.toString(), equalTo("Hello ΛRROW Meta!\n"))
+      Assertions.assertEquals(out.toString(), "Hello ΛRROW Meta!\n")
```

Comment:
```suggestion
      Assertions.assertEquals("Hello ΛRROW Meta!\n", out.toString())
```

A tiny change that impacts on test message errors: https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertEquals(java.lang.Object,java.lang.Object)

Note: I try to avoid JUnit assertions in favor of AssertJ or Hamcrest to avoid these failures.